### PR TITLE
 Added logic for automatic interface configuration

### PR DESCRIPTION
**Pull request by xnojack**
For simple ipv4 cidrs, added logic to automatically configure the interface. It'll look for an interface that matches the IP address and CIDR range given. Allowing for more dynamic deployments in modern Debian systems.

Also adjusted interface filters to match current docker interface names.

### CODE REVIEW

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -85,11 +85,11 @@ jobs:
...
-      - name: Login to DockerHub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      # - name: Login to DockerHub
+      #   uses: docker/login-action@v3
+      #   with:
+      #     username: ${{ secrets.DOCKERHUB_USERNAME }}
+      #     password: ${{ secrets.DOCKERHUB_TOKEN }}
```
Comment (shawly) [inline .github/workflows/docker-publish.yml]: This is necessary, please remove this commit or else I can't merge your PR

**Comment by shawly** [comment]
Also you should add some documentation to the KEEPALIVED_INTERFACE var about this feature in the README.md

File: .github/workflows/docker-publish.yml (hunk 1 above)
Comment (xnojack) [reply .github/workflows/docker-publish.yml]: Sorry about that. My intention was only the first 2 commits. Its reverted.

**Comment by xnojack** [comment]
> Also you should add some documentation to the KEEPALIVED_INTERFACE var about this feature in the README.md

Added some to the environment variable table.

**Pull request closed without merging**